databroker/src/grpc/kuksa_val_v1/val.rs

```diff
@@ -792,7 +792,73 @@ fn proto_entry_from_entry_and_fields(
         }
         if all || fields.contains(&proto::Field::MetadataValueRestriction) {
             metadata_is_set = true;
-            // TODO: Add to Metadata
+            metadata.value_restriction = match entry.metadata().allowed.as_ref() {
+                Some(allowed) => match allowed {
+                    broker::DataValue::StringArray(vec) => Some(proto::ValueRestriction {
+                        r#type: Some(proto::value_restriction::Type::String(
+                            proto::ValueRestrictionString {
+                                allowed_values: vec.clone(),
+                            },
+                        )),
+                    }),
+                    broker::DataValue::Int32Array(vec) => Some(proto::ValueRestriction {
+                        r#type: Some(proto::value_restriction::Type::Signed(
+                            proto::ValueRestrictionInt {
+                                allowed_values: vec.iter().cloned().map(i64::from).collect(),
+                                min: None, // TODO: Implement
```

Comment:
So for now just creating a placeholder for supporting min/max?

Reply:
I have no strong opinion, depends on how much time that would take I think. If it is quick go for it, if not keep the TODOs and do it later.